colosseum/wrappers.py

```diff
@@ -37,3 +40,66 @@ def horizontal(self):
     def vertical(self):
         """Return the vertical border spacing."""
         return self._horizontal if self._vertical is None else self._vertical
+
+
+class Uri:
+    """Wrapper for a url."""
+
+    def __init__(self, url):
+        self._url = url
+
+    def __repr__(self):
+        return 'url("%s")' % self._url
```

Comment:
@freakboy3742 what should be the canonical url repr?
I just used `'url("%s")'` but it could be

`"url('%s')"` or `'url(%s)'`

Reply:
Given the context, I'd say that the quotes, while allowed by the standard, are redundant, so I'd leave them off the canonical form.

Reply:
True, but given:

>Some characters appearing in an unquoted URI, such as parentheses, white space characters, single quotes (') and double quotes ("), must be escaped with a backslash so that the resulting URI value is a URI token: '\(', '\)'.

I felt was easier to simply quote and avoid having to add escapes. Thoughts?

Reply:
Ok - so I can see two alternatives here.

Firstly, we could consider ignoring `"`, `'` and `()` characters entirely. Have you included these characters because it makes sense to include them, or because the standard actually requires them? Does the URL have to be URL encoded? Are escaped quote characters actually a risk?

If we *can't* ignore them, we should be formatting with `%r` instead of `%s`.  That will include the external quotes, but will also autoescape the content of the string, including switching quotes between `'` and `"` such that quoting will be minimized.